Offers about and paths ready

kamilkoterba opened the pull request.

kamilkoterba committed:
Removing unused fallbacks

kamilkoterba committed:
Ignoring grunt-icon task lefftovers

kamilkoterba committed:
Reorganising template structure for some reusable parts

kamilkoterba committed:
Removing unused svg files

kamilkoterba committed:
Disabling roots sidebar for whole site

kamilkoterba committed:
Job offer single view template with example data

kamilkoterba committed:
Removing modal view from job offer list and adding link to single job offer page

kamilkoterba committed:
Job offers linking

kamilkoterba committed:
Fixed job offers previews

kamilkoterba committed:
Career paths on job offers list page

kamilkoterba committed:
Akismet plugin update

kamilkoterba committed:
Removing links to tag filters, minor fixes

kamilkoterba committed:
Filling main page with texts

kamilkoterba committed:
Adding paragraphs to even column breaking for Firefox

kamilkoterba committed:
Removing LinkedIn button for now till we develop connecting by LinkedIn

kamilkoterba committed:
Reduced height of header to show tab navigation on listing pages

kamilkoterba committed:
Parent page banner text change

kamilkoterba committed:
Adding regenerate thumbnails plugin

kamilkoterba committed:
Template for Life in Wikia page plus data retrival and styling for play with us tab

kamilkoterba committed:
Some grammar correction

kamilkoterba committed:
Display for regular article

kamilkoterba committed:
About Wikia - Wide Images and narrow paragraphs template

kamilkoterba committed:
Job offers translations, text corrections, switch off links to tags

kamilkoterba committed:
Retriving job offers for mainpage from database

kamilkoterba committed:
New recruitment banner with fussball

kamilkoterba committed:
Adding Fandom banner

kamilkoterba committed:
Switching banners on boostrap tabs hook on Lifie in Wikia page

kamilkoterba committed:
Career path template with templates for building hierarchial navigation

kamilkoterba committed:
Sidebar menu for career paths

kamilkoterba committed:
Quick fix to save proportions of images returned by wordpress editor

kamilkoterba committed:
Retriving job offers list on mainpage from widget - adjusting Flexi Pages widget to return Job Offers markup

kamilkoterba committed:
Adding constant  links in Career Paths main page section. Adding constant path to fetch featured career path.

kamilkoterba committed:
Application form modal on job offers pages

kamilkoterba committed:
Navigation minor enhacement

kamilkoterba committed:
Stretch images in benefits to 100%

kamilkoterba committed:
Minor style changes - footer buttons

kamilkoterba committed:
Fix for unexpected margin under flexslider

kamilkoterba committed:
Adding new job offers

kamilkoterba committed:
Updating job offers texts

kamilkoterba committed:
Job offer - Software test engineer template

kamilkoterba committed:
Adding QA lead job offer

kamilkoterba committed:
Universal template for Job offers, moving content to wordpress panel

kamilkoterba committed:
Job offers - Removing static content templates

kamilkoterba committed:
Revert "Akismet plugin update"

This reverts commit 8186a6475b41f6c4843a081cf17694658872fef2.

kamilkoterba committed:
Revert "Adding regenerate thumbnails plugin"

This reverts commit 931180f85eb537ed0330c6beea1f4cd5ec5b442e.

kamilkoterba committed:
Rendering HTML outside of PHP, changeing static content to dynamic message

kamilkoterba committed:
Adding news supptor to careers page

kamilkoterba committed:
Fix for not working redirect - buffering result

kamilkoterba committed:
Fix for broken link on job offers list page"

kamilkoterba committed:
404 page messages translations

kamilkoterba committed:
Prevent notices when empty post provided

kamilkoterba committed:
Getting rid of php notices

kamilkoterba committed:
Merge update conflicts from wikia-grid-implementation

kamilkoterba committed:
Fix main navigation after merge

kamilkoterba committed:
Changing text on banner

kamilkoterba committed:
Resolving conflicts for app.less

kamilkoterba committed:
Typography - body, titles and buttons font-size and line-heights

kamilkoterba committed:
Spaces to tabs after merge

kamilkoterba committed:
Spaces to tabs after merge

kamilkoterba committed:
Responsive typography for blockquote and subtext

kamilkoterba committed:
Adding spaces before and after elements for different resolutions

kamilkoterba committed:
Converting spaces to tabs

kamilkoterba committed:
Applying to Life in wikia subpages similar templates hierarchy like in Careers subpage

kamilkoterba committed:
Removing unused template - parent-page-life-in-wikia

kamilkoterba committed:
Fix for highlighting active tab in main menu

kamilkoterba committed:
Removing duplicated contact section

kamilkoterba committed:
Splitting app.less into smaller meaningful files

kamilkoterba committed:
Updating styles for Job offers list page

kamilkoterba committed:
Minor styles updates

kamilkoterba committed:
Adding tabs menu to news list

kamilkoterba committed:
Adding scrolling to content on tab click and scroll down button

kamilkoterba committed:
Merge branch 'offers-about-and-paths-ready' into responsive-typography-merge-test

Conflicts:
	wp-content/themes/wikia-careers/assets/less/app.less
	wp-content/themes/wikia-careers/assets/less/wikia-careers/mixins.less

kvas-damian commented on wp-content/themes/wikia-careers/assets/less/app.less:
```diff
@@ -217,6 +214,24 @@ html, body {
+	&.list-page-header {
+		@height-of-tab-nav: 70px;
+		min-height: calc(~"100% - @{site-header-lineheight} - @{height-of-tab-nav}");
+		min-height: -webkit-calc(~"100% - @{site-header-lineheight} - @{height-of-tab-nav}");
```
please use mixin when you need prefixes, It'd much easier in future to remove prefixes in one place.
In addition I found some `calc` calls without prefixes. Not sure if it was in this pull request.

kamilkoterba replied:
All browers except Balckberry browser works fine with just calc, so I skipped prefixed value.
http://caniuse.com/calc

kvas-damian commented on wp-content/themes/wikia-careers/assets/less/app.less:
```diff
@@ -519,9 +551,16 @@ div.wpcf7-response-output {
-	.strong;
+.news-feed a {
+	h3 {
+		.h4;
```
I think you created font-sizes definition to avoid lines like this.

kvas-damian commented on wp-content/themes/wikia-careers/assets/less/wikia-careers/job-offer-single.less:
```diff
@@ -0,0 +1,305 @@
+  }
+
+  /* Content - detailed job description */
+  .job-offer-details {
```
Please remove empty block.

kamilkoterba committed:
Refactoring CSS - adding icons classes to html to avoid copying icons code multiple times in compiled css file

kamilkoterba committed:
Merge pull request #7 from Wikia/sprint4-part3

Sprint4 part3

kamilkoterba committed:
Merge pull request #8 from Wikia/sprint4-less-refactoring

Splitting app.less into smaller meaningful files

kamilkoterba committed:
Moving .h1 class to html to avoid styles repetition

kamilkoterba committed:
Moving .padding-section-vertical class to html to avoid styles repetition

kamilkoterba committed:
Merge pull request #9 from Wikia/sprint4

Sprint4

kamilkoterba committed:
Removing empty css block, moving .h4 class to HTML to aviod code repetition

kamilkoterba committed:
Merge pull request #10 from Wikia/responsive-typography-merge-test

Responsive typography merge test

kamilkoterba committed:
Adding arrow down to main page plus minor changes indents and properties order

kamilkoterba committed:
Refactoring header banners for unified height

kamilkoterba committed:
Minor fix doubled line, lighter dim on banner

kamilkoterba committed:
Adjusting titles margins to vertical grid

kamilkoterba committed:
Fix for buttons height

kamilkoterba committed:
Adding bootstrap popover manual trigger

hakubo commented on wp-content/themes/wikia-careers/templates/job-offers/job-offer-single-head.php:
```diff
@@ -0,0 +1,55 @@
+			<button type="button" class="go-back icon-arrow-left-gray" data-dismiss="modal" aria-hidden="true">Powrót</button>
+			<div class="modal-header">
+				<h2 class="modal-title" id="myModalLabel"><?php echo $post->post_title ?></h2>
+<!--				<p class="short-descr">Lokalizacja lub nazwa zespołu</p>-->
```
Does this need to be here?
same question for all the comments here

hakubo commented on wp-content/themes/wikia-careers/templates/job-offer-single.php:
```diff
@@ -0,0 +1,65 @@
+			<?php while (have_posts()) : the_post(); ?>
+				<?php the_content(); ?>
+			<?php endwhile; ?>
+
```
why so many \n?

hakubo commented on wp-content/themes/wikia-careers/templates/job-offer-single.php:
```diff
@@ -0,0 +1,65 @@
+	<div class="row">
+		<div class="col-xxs-4 col-xs-4 col-sm-6 col-md-12 col-lg-10 col-lg-offset-1 job-offer-details">
+
+			<?php while (have_posts()) : the_post(); ?>
```
couldn't that be in one <?php?

hakubo commented on wp-content/themes/wikia-careers/templates/job-offer-single.php:
```diff
@@ -0,0 +1,65 @@
+			<ul class="tags">
+				<?php foreach (get_the_tags($post->ID) as $tag) : ?>
+					<li>
+						<span><?php echo $tag->name ?><span>
```
<?= $tag->name ?>

hakubo commented on wp-content/themes/wikia-careers/templates/content-single.php:
```diff
@@ -1,14 +1,34 @@
+
+	<?php wp_link_pages(array('before' => '<nav class="pagination">', 'after' => '</nav>')); ?>
+
+<?php endwhile; ?>
```
\n

hakubo commented on wp-content/themes/wikia-careers/templates/content-single.php:
```diff
@@ -1,14 +1,34 @@
+		<div class="row">
+			<div class="col-xxs-2 col-xs-2 col-sm-3 col-md-6 col-lg-5 col-lg-offset-1">
+				<?php if ( get_option('page_for_posts' ) ) : ?>
+					<a href="<?php  echo get_permalink( get_option('page_for_posts' ) ); ?>" class="go-back icon-arrow-left-gray">Powrót do listy aktualności</a>
```
\s\s

hakubo commented on wp-content/themes/wikia-careers/templates/content-page.php:
```diff
@@ -1,4 +1,39 @@
+	}
+
+	?>
+
```
\n

hakubo commented on wp-content/themes/wikia-careers/templates/contact-section.php:
```diff
@@ -0,0 +1,6 @@
+		<h2>Chcesz o coś zapytać? Śmiało!</h2>
+		<a href="mailto:hr-pl@wikia-inc.com" class="btn-wikia-white btn-envelope">Wyślij e-mail</a>
+	</div>
+</section>
```
\n

hakubo commented on wp-content/themes/wikia-careers/templates/career-paths-section.php:
```diff
@@ -0,0 +1,53 @@
+$featured_page = get_page_by_path('kariera/sciezki-rozwoju/jakub/');
+
+
+//$pagees = get_pages(array(
```
// ?
why not to remove this code?

hakubo commented:
please check your white spaces
and new lines
many places where something is off

hakubo commented on wp-content/themes/wikia-careers/lib/cleanup.php:
```diff
@@ -203,7 +203,7 @@ function roots_remove_self_closing_tags($input) {
 add_filter('get_avatar',          'roots_remove_self_closing_tags'); // <img />
 add_filter('comment_id_fields',   'roots_remove_self_closing_tags'); // <input />
-add_filter('post_thumbnail_html', 'roots_remove_self_closing_tags'); // <img />
+//add_filter('post_thumbnail_html', 'roots_remove_self_closing_tags'); // <img />
```
remove dead code

hakubo commented:
Please create feature branches and ask often for code review,
the size (and in many places lack of context) of this PR does not allow me to give more meaningful CR

kamilkoterba commented:
@hakubo I addressed all your comments in my current branch share-icons
https://github.com/Wikia/careers-page/commit/c40ef42c698e9574779423e36d24e6818491e972
so it will go with next PR.

I also skipped note about using php echo short notation as whole project uses full notation and as we talked it's not defined in our coding conventions

kamilkoterba commented:
and sorry for large PR. Originally it was spread to 5 PRs, but I merged it after first round of CR to avoid further conflicts.

kvas-damian commented:
@kamilkoterba it's easier to verify fixes if you put them in branch that is currently under code review.

kamilkoterba commented:
I know, just wanted to avoid conflicts as all are minor and non blocking changes.
Already discussed with @hakubo 
merging.
